api/src/Entity/ContentNode.php

```diff
@@ -89,6 +91,13 @@ class ContentNode extends BaseEntity implements BelongsToCampInterface {
     #[Groups(['read', 'write'])]
     public ?ContentNode $parent = null;
 
+    /**
+     * The prototype ContentNode from which the content is copied during creation.
+     */
+    #[ApiProperty(example: '/content_nodes/1a2b3c4d')]
+    #[Groups(['Default', 'contentNode:update'])] // TODO: allow in create schema only
+    public ?ContentNode $prototype = null;
```

Comment:
Müsste das noch in die DB gespeichert werden?

Reply:
Hatten wir bisher im alten Backend nicht in der DB.